org.diffoscope.Diffoscope.yaml

```diff
@@ -0,0 +1,61 @@
+app-id: org.diffoscope.Diffoscope
```

Comment:
Please reformat the file to follow our [style guide](https://github.com/flathub/flathub/wiki/YAML-Style-Guide).